tests/js/panelsTests.js

```diff
@@ -362,6 +345,70 @@ https://github.com/gpii/universal/LICENSE.txt
         jqUnit.assertEquals("The decrease button should have the correct enabled/disabled state", decreaseDisabled, that.locate("decrease").prop("disabled"));
     };
 
+    /*************************
+     * Text Size Panel Tests *
+     *************************/
+
+    fluid.defaults("gpii.tests.firstDiscovery.panel.textSize", {
+        gradeNames: ["gpii.firstDiscovery.panel.textSize", "autoInit"],
+        messageBase: {
+            rangeInstructions: "Adjust the text and controls to a size you like best.",
+            maxLabel: "max",
+            minLabel: "min",
+            increaseLabel: "larger",
+            decreaseLabel: "smaller"
+        },
+        model: {
+            value: 1
+        },
+        modelListeners: {
+            // rerenders on modelChange like panel behaves in the prefsEditor
+            "value": "{that}.refreshView"
+        }
+    });
+
+    fluid.defaults("gpii.tests.textSizePanel", {
+        gradeNames: ["gpii.tests.rangePanel", "autoInit"],
+        components: {
+            range: {
+                type: "gpii.tests.firstDiscovery.panel.textSize",
+                container: ".gpiic-fd-textSize"
+            }
+        }
+    });
+
+    /***************************
+     * Speech Rate Panel Tests *
+     ***************************/
+
+    fluid.defaults("gpii.tests.firstDiscovery.panel.speechRate", {
+        gradeNames: ["gpii.firstDiscovery.panel.speechRate", "autoInit"],
+        messageBase: {
+            rangeInstructions: "Adjust the speed at which items on the screen are read out loud.",
+            maxLabel: "fast",
+            minLabel: "slow",
+            increaseLabel: "faster",
+            decreaseLabel: "slower"
+        },
+        model: {
+            value: 1
+        },
+        modelListeners: {
+            // rerenders on modelChange like panel behaves in the prefsEditor
+            "value": "{that}.refreshView"
+        }
+    });
+
+    fluid.defaults("gpii.tests.speechRatePanel", {
+        gradeNames: ["gpii.tests.rangePanel", "autoInit"],
```

Comment:
I think it is reasonable to rerun the base grade tests for the derived grade, especially since we have taken care to make sure that all we need to do is add a grade name to the derived grade's tester. This issue also came up with @simonbates "speaking keyboard panel" and we also considered there that verifying the base grade contract was useful to do. It's one of the strengths of the IoC testing system that composition of test case grades can mirror composition of the grades under test, meaning that it takes almost no extra work to ensure that composition has not broken invariants.